Update B2B sample readme (#423)
wso2/samples-is: b2b-sample/README.md

## Guardio Insurance

**Guardio** is an automobile insurance service company that provides its services to other business organizations such as automobile lenders
through multiple business applications.

**Guardio-Business-App** and **Guardio-Admin-App** are two such applications.

### Guardio-Business-App

**Guardio-Business-App** is the application that provides insurance and claims settlement capabilities for other
businesses, so that those businesses can use the software for internal requirements.

**Best Car Mart** is one of the customer of **Guardio** Insurance.
**Best Car Mart** organization's staff needs to allow corporate users of their organization to log
into the **Guardio-Business-App** application using their corporate SSO system, to get used to the above-mentioned
capabilities.
To do that, they will configure the **Guardio-Business-App** application to leverage their corporate
IdP for authentication. This will be setup through the **Guardio-Admin-App**.

### Guardio-Admin-App

**Guardio-Admin-App** is the application that provides administrative capabilities to administrators of **Guardio** customer/partner organizations.

> The provided provides administrative capabilities are managing users, assigning roles,

Reviewer: ```suggestion
The provided administrative capabilities are managing users, assigning roles,
```

> configuring an identity provider for the organization, and customize the login flow of **Guardio-Business-App** business application for your organization.

Reviewer: ```suggestion
configuring an identity provider for the organization, and customize the login flow of **Guardio-Business-App** business application for their respective organization.
```

>| SCIM2 Roles API |                            `View Role`, `Update Role`                            |
>| SCIM2 Groups API           |                           `View Group`, `Update Group`                           |
>| Application Management API           |                     `View Application`, `Update Application`                     |
>| Identity Provider Management API           |      `View Identity Provider`, `Create Identity Provider`, `Update Identity Provider`, `Delete Identity Provider`       |

![API Authorization](libs/business-admin-app/ui/ui-assets/src/lib/images/guardio-admin-app-api-authorization.png)

### Step 4: Create roles for the Guardio-Admin-App

- On the root organization's Console, go to User Management > Roles.
- Create a new role with following information.
>| Field                            |                                                                       Value(s)                                                                        |
>|----------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------:|

> >| Role Name                        |                                                                `Guardio Administrator`                                                                |

Reviewer: ```suggestion
>| Role Name                        |                                                                `Guardio-Administrator`                                                                |
```

Can we create role with space in the middle?

>| Select the role audience                  |                                                                     `Application`                                                                     |
>| Assigned application                |                                                                  `Guardio-Admin-App`                                                                  |
>| Select API Resource | Listed all permissions of `SCIM2 Users API `, `SCIM2 Roles API`, `SCIM2 Groups API`, `Application Management API`, `Identity Provider Management API` |

Role basic information:
![Role Basic Info](libs/business-admin-app/ui/ui-assets/src/lib/images/admin-role-basic-info.png)

Role's permission selection:
![Admin Role Permissions](libs/business-admin-app/ui/ui-assets/src/lib/images/admin-role-permissions.png)

```agsl
Note
   
   When you create an application audience role, it will be associated with the selected application. Also that application can use only application audienced roles.

>    Instead, you can change the allowed audience for role association from the application configuration and use roles created in organization audience.

Reviewer: ```suggestion
   Instead, you can change the allowed audience to organization audience for role association from the application configuration and use roles created in organization audience.
```
